src/forms/BookingTimeForm/FieldDateAndTimeInput.js

```diff
@@ -28,7 +29,12 @@ import NextMonthIcon from './NextMonthIcon';
 import PreviousMonthIcon from './PreviousMonthIcon';
 import css from './FieldDateAndTimeInput.module.css';
 
-const MAX_TIME_SLOTS_RANGE = 180;
+// A maximum number of days forwards during which a booking can be made.
+// This is limited due to Stripe holding funds up to 90 days from the
+// moment they are charged. Also note that available time slots can only
+// be fetched for 180 days in the future.
```

Comment:
Could you also update the comment here (and in config.js) about time-slot query:
https://www.sharetribe.com/api-reference/marketplace.html#query-time-slots

It seems that the 180 day limit has been pushed forward.